akka-remote/src/main/resources/reference.conf

```diff
@@ -988,6 +989,23 @@ akka {
         # of a network partition that you need to survive.
         give-up-system-message-after = 6 hours
 
+        # Outbound streams are stopped when they haven't been used for this duration.
+        # They are started again when new messages are sent.
+        stop-idle-outbound-after = 5 minutes
+
+        # Outbound streams are quarantined when they haven't been used for this duration
+        # to cleanup resources used by the association, such as compression tables.
+        # This will cleanup association to crashed systems that didn't announce their
+        # termination.
+        # The value should be longer than the length of a network partition that you
+        # need to survive. The value must also be greater than stop-idle-outbound-after.
```

Comment:
I think this needs to be 10x greater than the stop-idle-outbound-after otherwise this hand shake will keep the outbound lanes from every been shut down.

Reply:
Are you sure? it was long time since I worked on this, but I think the low frequency handshakes (quarantine-idle-outbound-after / 10) don't touch the timestamp when they are sent, but when the reply is received.

Even though that could keep the connection alive if the systems are alive, and replies, I don't see that as a big problem. The connection will be woken up at quarantine-idle-outbound-after / 10 no matter. Low values of quarantine-idle-outbound-after are not recommended, but I don't think we have to enforce some value.

Perhaps you don't want to stop idle at all, but still want this feature.

Reply:
Yes I accidently came across this when trying different versions of each timeout.

Ah yes i see if we put that limit in we would be disabling users from using this feature but not stopping idle via `stop-idle-outbound-after`. 

I still find it surprising in the case that I want to clean up TCP connections if there are no user messages and want this new feature I can accidently disable the `stop-idle-outbound-after` via the value of `quarantine-idle-outbound-after`

Reply:
Ok, we can add that 10x requirement, but then also support `off` as a possible value in `stop-idle-outbound-after`.

Reply:
or just add a note to the reference.conf (not worth putting too much effort into this)

Reply:
:+1: note is fine